### DISCUSSION TITLE
Are there files that are created? If so, where are they stored? Also, what files can I delete if I uninstall?

### DISCUSSION THREAD

**Discussion by nothing2obvi** [intro-skipper/intro-skipper#407] [Q&A] [answered]
I'm just wondering in case:

1.) I need to move the files to an external hard drive since the SSD in my Mac is running out of storage.

2.) I want to uninstall the plugin. Besides uninstalling it in the web UI, what files can I delete?

**Comment by AbandonedCart** [comment, marked as answer]
Go to the bottom of the plugin settings to edit timestamps and fingerprints. Check the option for cached files (each time) before clicking the erase buttons intro, recap, credits, previews. If it isn’t automatically deleted, you can delete Jellyfin/data/introskipper

That’s pretty much it. Best of luck

**Reply by nothing2obvi** [reply]
Thanks so much!
